cmd/podman/containers/ps.go

```diff
@@ -93,6 +94,7 @@ func listFlagSet(cmd *cobra.Command) {
 	flags.Bool("noheading", false, "Do not print headers")
 	flags.BoolVarP(&listOpts.Size, "size", "s", false, "Display the total file sizes")
 	flags.BoolVar(&listOpts.Sync, "sync", false, "Sync container state with OCI runtime")
+	flags.BoolVar(&listOpts.Restart, "restart", false, "Display the container restart count")
```

Comment:
I really do not like these flags, we already offer a complete customizable format with `--format` so I do not want to add flags for each option. This starts to add up quickly.

Reply:
I share the feeling, @Luap99.  In fact, I do not see any reason to add this kind of information to `ps` at all.  The `inspect` commands in combination with `--format` should be used for that.

See https://github.com/containers/podman/pull/17830/files#diff-a06598ad58c5be472e9e040f89c040b4548837ce23c60fdb4eba27ba766c571fR189-R190 as an example.

Reply:
Removed the flag.